tools/accuracy_checker/accuracy_checker/representation/text_detection_representation.py

```diff
@@ -58,5 +58,5 @@ def remove(self, indexes):
         self.description = np.delete(self.description, indexes)
 
 
-class TextDetectionPrediction(TextDetectionRepresentation):
+class TextDetectionPrediction(TextDetectionAnnotation):
```

Comment:
please stay base class `TextDetectionRepresentation`, just move description to the base class.
inheritance is now logically incorrect, predicted object should no be treated as annotation